content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -74,7 +74,7 @@ different Kubernetes components.
 | `CSIInlineVolume` | `true` | Beta | 1.16 | - |
 | `CSIMigration` | `false` | Alpha | 1.14 | 1.16 |
 | `CSIMigration` | `true` | Beta | 1.17 | |
-| `CSIMigrationAWS` | `false` | Alpha | 1.14 | |
+| `CSIMigrationAWS` | `false` | Alpha | 1.14 | 1.16 |
 | `CSIMigrationAWS` | `false` | Beta | 1.17 | 1.22 |
 | `CSIMigrationAWS` | `true` | Beta | 1.23 | |
```

Comment:
nit: Is there a second Beta version of `CSIMigrationAWS`?

Reply:
Yes. In second "sub-phase", the gate is defaulted to `true`. That is something I don't like. It sounds like that being 'Beta' is not guaranteed to be defaulted to `true`. What is making things more complicated is that the community has agreed to [default 'Beta' APIs to `false`](https://github.com/kubernetes/enhancements/pull/3137).

Reply:
New beta APIs will be default to false but a second beta version of an existing beta API will still default to true.
This was summarized in today's Kubernetes Community Meeting, I will update this comment with the link to the YouTube video when the video is up

Reply:
That new default nudges me to consider rethinking a [generated](https://github.com/kubernetes/website/pull/28036) feature gates page, done so that we can separate the rendering from the data.

We could even provide this page just as it is now, along with a new page that shows the same data quite differently.

Reply:
Link to the [Kubernetes Community meeting discussion](https://youtu.be/m1nNW7gnbU0?t=1252) about new beta APIs will be default false